ruka/cnpg/cnpg-scheduledbackups.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: cnpg-backup-scheduled
   namespace: cloudnativepg
 spec:
-  schedule: "0 12,21 * * * *" #8AM CLT - 5PM CLT (after nightwork and daywork at the summit)
+  schedule: "0 0 12,21 * * *" #9AM CLT - 6PM CLT (after nightwork and daywork at the summit)
```

Comment:
This is UTC time, it will not work at specified CLT during CLST so better labeling as UTC

Reply:
its really both, the comment is to quickly check that the time on the cronjob (12 and 21) correspond to 9am-6pm CLT, but i can leave the time in UTC on the comment also